pkgs/top-level/all-packages.nix

```diff
@@ -159,6 +159,8 @@ in
 
   castxml = callPackage ../development/tools/castxml { };
 
+  cdetheme = callPackage ../data/themes/cdetheme { };
```

Comment:
Actually, FWIW it seems like a lot of the theme packages have the suffix `-gtk-theme`, so maybe that would make sense here as well.

Reply:
@sauyon Done, thanks!

Reply:
In general I prefer to use the upstream project name if that is reasonable. It would make it easier to find the package when someone searches for it in NixOS.

Reply:
@romildo There is no common name really. They call tarball "cdetheme" on gnome-look, repo "cde-motif-theme" on github, "CDE / Motif theme" in titles.